mne/io/meas_info.py

```diff
@@ -1681,7 +1682,11 @@ def create_info(ch_names, sfreq, ch_types=None, montage=None):
         raise ValueError('ch_types and ch_names must be the same length '
                          '(%s != %s)' % (len(ch_types), nchan))
     info = _empty_info(sfreq)
-    info['meas_date'] = np.array([0, 0], np.int32)
+    # The minium timestamp in Windows is 86400
+    if 'Windows' in platform.system():
+        info['meas_date'] = np.array([86400, 86400], np.int32)
+    else:
+        info['meas_date'] = np.array([0, 0], np.int32)
```

Comment:
Instead of triaging based on system, let's just always do `86400, 86400`. It's better if we don't have values based on the system someone is using.